packages/base/src/types/ValueState.js

```diff
@@ -8,6 +8,7 @@ const ValueStates = {
 	Success: "Success",
 	Warning: "Warning",
 	Error: "Error",
+	Information: "Information",
```

Comment:
If you provide a new value state, you should also implement the visual design (not only of the valueStateMessage) of the ui5-input itself, because now the users set "Information", previously not.